osf/management/commands/check_ia_metadata.py

```diff
@@ -0,0 +1,104 @@
+import re
+import requests
+from django.core.management.base import BaseCommand
+from osf.models import Registration
+from website import settings
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class IAMetadataError(Exception):
+    def __init__(self, message=None, fields=None):
+        self.message = message
+        self.fields = fields
+
+
+get_ia_field = lambda field: Registration.IA_MAPPED_NAMES.get(field, field)
+
+mirrored_attrs = list(Registration.SYNCED_WITH_IA)
+from django.db.models import F
+
+mirrored_fields = mirrored_attrs + ['subjects', 'tags', 'affiliated_institutions']
+mirrored_relationships = [
+    'subjects__text',
+    'affiliated_institutions__name',
+    'tags__name',
+]
+
+
+def check_ia_metadata(collection=settings.IA_ROOT_COLLECTION, guids=None):
```

Comment:
See previous suggestion to use a time delta and filter out only registrations with updated fields node logs -- this should be a follow-up at this point.